web/src/features/map/MapTooltip.tsx

```diff
@@ -92,26 +68,24 @@ export function TooltipInner({
 
 function DataValidityBadge({
   hasOutage,
-  estimationMethod,
-  estimatedPercentage,
+  estimated,
 }: {
-  hasOutage: boolean | undefined;
-  estimationMethod: string | undefined;
-  estimatedPercentage: number | undefined;
+  hasOutage?: boolean | null;
+  estimated?: number | boolean | null;
```

Comment:
I think this idea of a property that can be both a number and a boolean is going to cause problems down the road, but we can reconsider this at a later point :)

Reply:
Possibly but considering what we learned about the estimations recently I think that maybe it should just be a number for all time aggregations.